vite.config.ts

```diff
@@ -24,8 +24,13 @@ export default defineConfig({
     // others are defaults from emscripten and vite
     target: ['es2020', 'edge88', 'firefox68', 'chrome75', 'safari13'],
     lib: {
-      entry: resolve(__dirname, 'src/index.ts'),
-      name: 'VueQrcodeReader'
+      entry: {
+        'vue-qrcode-reader': resolve(__dirname, 'src/index.ts')
+      },
+      formats: ['es', 'umd'],
+      name: 'VueQrcodeReader',
+      fileName: (format, entryName) =>
+        format === 'es' ? `${entryName}.js` : `${entryName}.${format}.js`
```

Comment:
Thanks for finding a proper solution to my cheap-ass workaround in the `package.json`. However, that means we don't have a `dist/vue-qrcode-reader.umd.cjs` anymore, right? I guess strictly speaking this breaks backwards compatibility. Is it possible to also generate a copy of `dist/vue-qrcode-reader.umd.js` that is named `dist/vue-qrcode-reader.umd.cjs` ?

Reply:
![Screenshot_2023-10-24-21-58-54-87_4d38fce200f96aeac5e860e739312e76.jpg](https://github.com/gruhn/vue-qrcode-reader/assets/10386119/e52a4e50-1edd-4e21-a76c-1b7fe011f62c)

It shouldn't break any things if users are using pinned versions. And jsdelivr (or maybe other CDNs too) seems to have a fallback mechanism when a file is no longer served in the upgraded version.

Reply:
Ok yeah that's good enough for this edge case :+1: